cg/cli/upload/vogue.py

```diff
@@ -132,6 +137,8 @@ def bioinfo(context, case_name, cleanup, target_load, dry):
         load_bioinfo_raw_inputs["analysis_workflow_name"],
         load_bioinfo_raw_inputs["analysis_workflow_version"],
     ) = _get_analysis_workflow_details(context, case_name)
+    if load_bioinfo_raw_inputs["analysis_workflow_name"] not in valid_workflows:
```

Comment:
I had a really hard time understanding the above two lines. Could you do something like:

```python
workflow_name, workflow_version = _get_analysis_workflow_details(context, case_name)
if workflow_name not in valid_workflows:
    raise AnalysisUploadError(f"Case upload failed: {case_name}. Reason: Bad workflow name.")
load_bioinfo_raw_inputs["analysis_workflow_name"] = workflow_name
load_bioinfo_raw_inputs["analysis_workflow_version"] = workflow_version
```

Reply:
I had it like this before, but changed it for sake of fewer lines. I'll do it this way. 👍

Reply:
I prefer readability over compactness!